projects/restaking-pool/contracts/RestakingPool.sol

```diff
@@ -459,6 +426,13 @@ contract RestakingPool is
         );
     }
 
+    function startWithdrawalCheckpoint(
+        string memory provider
+    ) external onlyOperator {
+        IEigenPod restaker = IEigenPod(_getRestakerOrRevert(provider));
+        restaker.startCheckpoint(true);
```

Comment:
what does true mean?

Reply:
it means to "force" checkpoint creation if there is any alredy started checkpoint